runtime/src/bank.rs

```diff
@@ -506,7 +506,7 @@ impl Bank {
         );
 
         self.capitalization.fetch_add(
-            (validator_rewards + storage_rewards) as usize,
+            (validator_rewards + storage_rewards) as u64, // WIP: why is it ok to cast f64 => u64?
```

Comment:
We don't care about fractional lamports, I think that's why

Reply:
yep

Reply:
Thanks for the confirmation! I'm now sure that this is safe with my current understanding of the codebase too!